scripts/check-binary.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+BASE_URL="https://binaries.soliditylang.org/bin"
+REPO_ROOT="$(dirname "$0")/.."
+LIST_FILE=/tmp/list.json
```

Comment:
Safer to ask the OS to generate the file. Temp dir might not be `/tmp` or `list.json` might already exist there.
```suggestion
LIST_FILE=$(mktemp -t solc-bin-list-XXXXXX.json)
```

Reply:
Oh, I did not know about `mktemp`, nice!